package.json

```diff
@@ -2,16 +2,23 @@
   "dependencies": {
     "@cljs-oss/module-deps": "^1.1.1",
     "@fluree/sjcl": "^1.0.8-3",
+    "axios": "^0.21.4",
+    "bufferutil": "^4.0.3",
     "js-sha3": "^0.8.0",
-    "scrypt-js": "^2.0.4",
-    "source-map-support": "^0.5.19"
+    "scrypt-js": "^3.0.1",
+    "source-map-support": "^0.5.19",
+    "utf-8-validate": "^5.0.5",
+    "ws": "^8.2.1"
+  },
+  "browser": {
+    "ws": false
   },
   "devDependencies": {
-    "karma": "^6.3.4",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-cljs-test": "^0.1.0",
-    "webpack": "^5.44.0",
-    "webpack-cli": "^3.3.12"
+    "karma": "6.3.4",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-cljs-test": "0.1.0",
+    "webpack": "5.51.1",
+    "webpack-cli": "4.8.0"
```

Comment:
Do we not want the `^` qualifier? I don't have a strong opinion either way, just curious.

Reply:
I removed it while troubleshooting to make sure I was getting the exact version I was asking for, and I generally prefer pinning to a specific version. But I don't think it affects anything concretely here, honestly.